package.json

```diff
@@ -74,7 +74,7 @@
     "react-styleguidist": "^5.0.6",
     "svgo": "^0.6.1",
     "webpack": "^2.6.1",
-    "webpack-dev-server": "^1.14.0"
+    "webpack-dev-server": "^3.1.14"
```

Comment:
@AleshaOleg does the `start` script make sense? I never used GrapeBrowser independently. If it's not used anywhere we could even remove `webpack-dev-server`.

My concern with simply upgrading it is that it won't work with webpack 2.6.1, but haven't tested that.

Reply:
@nikgraf I also never used this script. Just checked, I think this script supposed to use for this file - https://github.com/ubergrape/grape-browser/blob/master/examples/grape-input/index.html. But I dunno, never opened it too :( I think we can delete this file, and script with `webpack-dev-server` package.

Reply:
sounds good!